### PR TITLE
Add getter method for verifySSL

### PR DESCRIPTION
**Pull request by pprkut**
This is useful for other SSL-capable drivers that could do verification.

### CODE REVIEW

File: tests/unit/ClientTest.php [hunk 1]
```diff
@@ -174,9 +182,9 @@ protected function doTestResponse($responseFile, array $result, $method)
      *
      * @return Client
      */
-    protected function getClient($username = '', $password = '',  $ssl = false, $timeout = 0)
```
Comment (gianarb) [inline tests/unit/ClientTest.php]: This is a BC Break, I can not merge this PR.
Can you swap the parameters? 

```php
$username = '', $password = '',  $ssl = false, $timeout = 0, $verifySSL = false
```
In this way can work :) 

THanks a lot!

Comment (pprkut) [inline tests/unit/ClientTest.php]: I was not aware that BC was an issue with the Test classes.
I don't mind changing it, but want to make sure it's really necessary first.

File: tests/unit/ClientTest.php [hunk 2]
```diff
@@ -144,11 +145,18 @@ public function testFactoryMethod()
 
     public function testTimeoutIsFloat()
     {
-        $client =  $this->getClient('test', 'test', false, 0.5);
+        $client =  $this->getClient('test', 'test', false, false, 0.5);
 
         $this->assertEquals(0.5, $client->getTimeout());
     }
 
+    public function testVerifySSLIsBoolean()
```
Comment (gianarb) [inline tests/unit/ClientTest.php]: Can you enforce this assertion to `testVerifySSLIsTrue`? I know that other tests are like your but you know :) Make it better is good..

Thanks a lot

File: tests/unit/ClientTest.php (hunk 1 above)
Comment (gianarb) [inline tests/unit/ClientTest.php]: You are right.. I was totally out of scope. :) 
Sorry about that

File: tests/unit/ClientTest.php (hunk 2 above)
Comment (pprkut) [inline tests/unit/ClientTest.php]: Done :)

**Comment by pprkut** [comment]
Rebased onto master :)

**Comment by gianarb** [comment]
Thanks a lot! :) Sorry for the delay!

**Pull request merged**